documentation/source/user_section/installation.rst

```diff
@@ -80,9 +80,92 @@ recommend you download the Github version.
 Install on Windows 10 with WSL
 ------------------------------
 
-Windows subsystem for Linux (WSL) is available on Windows 10 and it makes it possible to run native Linux programs,
-such as SCT. Checkout the `installation tutorial for WSL <https://github.com/neuropoly/spinalcordtoolbox/wiki/SCT-on-Windows-10:-Installation-instruction-for-SCT-on-Windows-subsytem-for-linux>`_.
+Windows subsystem for Linux (WSL) is available on Windows 10 and it makes it possible to run native Linux programs, such as SCT.
 
+#. Install Windows Subsystem for Linux (WSL)
+
+   - Install `Xming <https://sourceforge.net/projects/xming/>`_.
+
+   - Install  `Windows subsystem for linux and initialize it <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_.
+
+       - Please install the Ubuntu 18.04 LTS distro.
+
+#. Environment preparation
+
+   Run the following command to install various packages that will be needed to install FSL and SCT. This will require your password
```

Comment:
I don't know which of the following packages were installed specifically for FSL. With FSLeyes, the packages listed in section 4 should be enough.

Reply:
we can now forget about FSL